docs/source/run-workflows/observe/observe.md

```
@@ -227,6 +228,13 @@ For complete information about developing and integrating custom telemetry expor
 
   :::
 
+  :::{tab-item} MLflow
+  :sync: MLflow
+
+    :::{include} ./observe-workflow-with-mlflow.md
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)